pkg/odo/cli/init/init.go

```diff
@@ -113,7 +114,7 @@ func (o *InitOptions) Validate() error {
 		return err
 	}
 
-	if len(o.flags) == 0 && log.IsJSON() {
+	if len(o.flags) == 0 && fcontext.GetOutput(ctx) {
```

Comment:
Would it make sense to have this function named `fcontext.IsJsonOutput`? Or make `GetOutput` return the actual output value as a constant?
(I am thinking of cases where we might need to support other output types in the future.)